CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to
 
 ## [Unreleased]
 
+## [1.2.5] - 2023-05-02
```

Comment:
Can you check the list of link targets in the bottom of this file adjust them?

Reply:
Thanks, totally missed those